### PR TITLE
feat/rework pass sync

### PR DESCRIPTION
**Pull request by krzysztofdrys**
- **feat: connect firebase sdk**
- **chore: rework endpoints for push**

### CODE REVIEW

File: internal/pass/sync/handlers.go [hunk 1]
```diff
@@ -26,8 +26,67 @@ func ExtensionRequestSync(syncingApp *Syncing) gin.HandlerFunc {
 			return
 		}
 
+		resp, err := syncingApp.RequestSync(gCtx, fcmToken)
+		if err != nil {
+			logging.Errorf("Failed to request sync: %v", err)
```
Comment (tobiaszheller) [inline internal/pass/sync/handlers.go:31]: yyou can use log from context here as well and everywhere in that handler

File: internal/pass/sync/handlers.go [hunk 2]
```diff
@@ -26,8 +26,67 @@ func ExtensionRequestSync(syncingApp *Syncing) gin.HandlerFunc {
...
+		token, err := tokenFromRequest(gCtx)
+		if err != nil {
+			log.Errorf("Failed to get token from request: %v", err)
+			gCtx.Status(http.StatusForbidden)
+			return
+		}
+		fcmToken, err := syncingApp.VerifyExtWaitForSyncToken(gCtx, token)
+		if err != nil {
+			log.Errorf("Failed to verify proxy token: %v", err)
+			gCtx.String(http.StatusUnauthorized, "Invalid auth token")
```
Comment (tobiaszheller) [inline internal/pass/sync/handlers.go:56]: i am not, do we want to return that info to user? maybe log is just enough

**Review by tobiaszheller — APPROVED** [review]
lgtm, left 2 comments

File: internal/pass/sync/handlers.go (hunk 1 above)
Comment (krzysztofdrys) [reply internal/pass/sync/handlers.go]: done, thanks

**Pull request merged**